cainafrica/src/components/Footer.js

```diff
@@ -84,43 +84,73 @@ const Footer = () => {
+                     <p style={{ marginTop: 0, marginBottom: "8px" }}>+2347049879535</p>
+                     <div className="social-icons" style={{ display: 'flex', alignItems: "center"}}> 
+                    <InlineFollowButtons //From sharethis-reactjs Library
                         config={{
-                        action: 'Follow us:', // call to action (STRING)
-                        action_enable: false,  // show/hide call to action (true, false)
-                        action_pos: 'bottom', // position of call to action (left, top, right)
-                        alignment: 'center',  // alignment of buttons (left, center, right)
-                        color: 'social',       // set the color of buttons (social, white)
-                        enabled: true,        // show/hide buttons (true, false)
-                        networks: [           // which networks to include (see FOLLOW NETWORKS)
-                            'twitter',
-                            'facebook',
-                            'instagram',
-                            'linkedin',
-                            'youtube'
+                        action: "Follow us:", // call to action (STRING)
+                        action_enable: false, // show/hide call to action (true, false)
+                        action_pos: "bottom", // position of call to action (left, top, right)
+                        color: "social", // set the color of buttons (social, white)
+                        enabled: true, // show/hide buttons (true, false)
+                        // alignment: "",
+                        networks: [
+                            // which networks to include (see FOLLOW NETWORKS)
+                            "twitter",
+                            "facebook",
+                            "instagram",
+                            "linkedin",
+                            "youtube",
+                            'github',
                         ],
-                        padding: 8,           // padding within buttons (INTEGER)
-                        profiles: {           // social profile links for buttons
-                            twitter: 'cainafrica',
-                            facebook: 'cainafrica',
-                            instagram: 'cainafrica',
-                            linkedin: 'company/catering-to-africans-in-need/',
-                            youtube: '/channel/UCFI55OCmrLlN28M5wDPBNxQ'
+                        padding: 8, // padding within buttons (INTEGER)
+                        profiles: {
+                            // social profile links for buttons
+                            twitter: "cainafrica",
+                            facebook: "cainafrica",
+                            instagram: "cainafrica",
+                            linkedin: "company/catering-to-africans-in-need/",
+                            youtube: "/channel/UCFI55OCmrLlN28M5wDPBNxQ",
+                            github: "isibord",
```

Comment:
please remove github from the list

Reply:
I needed to add the whatsapp icon and there was not space for it, also InlineFollowButton for whatsapp was not working so i used github to create one more space so the wahtsapp icon could show, if i just added the whatsapp icon it would take over the youtube space.

![Screenshot from 2021-08-16 20-29-02](https://user-images.githubusercontent.com/55482979/129619322-edb5efa4-8a59-421b-86c8-3f6b3a5f6bf6.png)

Reply:
Got it. Could you rename it to placeholder? The reason is because it could make someone believe we list github on the website but actually don't